pulp_docker/app/tasks/sync_stages.py

```diff
@@ -21,17 +22,7 @@
 }
 
 
-class TempTag:
-    """
-    This is a pseudo Tag that will either become a ManifestTag or a ManifestListTag.
-    """
-
-    def __init__(self, name):
-        """Make a temp tag."""
-        self.name = name
-
-
-class TagListStage(Stage):
+class DockerFirstStage(Stage):
     """
     The first stage of a pulp_docker sync pipeline.
```

Comment:
Maybe it would be good to add a second paragraph that explains the steps that take place since this is an unusually heavy first stage.